pkgs/development/python-modules/find-libpython/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "find_libpython";
+  version = "0.3.0";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-bn/l2a9/rW3AZstVFaDpyQpx8f6yuy+OTNu0+DJ26eU=";
+  };
+
```

Comment:
The upstream repo contains [tests](https://github.com/ktbarrett/find_libpython/tree/master/tests). 

Please add `pythonImportsCheck`.